Assets/Galaxy/GalaxyPlaneMat.mat

```diff
@@ -197,7 +197,7 @@ Material:
       second: 1
     - first:
         name: _TransitionAlpha
-      second: 0.000013768673
```

Comment:
This change doesn't seem to have anything to do with your PR and is probably a side effect of Unity's editor. There are pending changes in the Phone branch that will help with Unity's editor and material modification. Please remove this file from your PR.